### ISSUE TITLE
FastCopy

### ISSUE THREAD

**Issue by dtgm** [teknowledgist/Chocolatey-packages#3]
I had intended to convert fastcopy into an automatic package, but only half finished it.

How do you feel about moving those packages over into my repo?

**Comment by dtgm** [comment]
Are these packages automatic? 

If not, I have the portable (easy) package ready to go I prefer we use.

The install package I could take as well, but updates to the script would require issuing PR against my repo as I would welcome the help of you continuing to help support the package. If you are against that, you should become the maintainer of fastcopy as well so you can manually keep them in sync.

Obviously, I would want you to be a maintainer of all 3 as well so you could help make updates and push the fixes for interim packages (between program version releases). BTW, you did a good job with fastcopy.install.

**Comment by teknowledgist** [comment]
Hi,

I haven't yet figured out how to do automatic packages, but I'd be happy to be a co-maintainer on the FastCopy packages so that I might learn.  Feel free to own the repository pointed to by the package.  I'll remove my FastCopy folder from my repository and just submit pull requests against yours.

Just to be asked is a compliment from someone near the top of the Chocolatey mountain.  :-)

**Comment by teknowledgist** [comment]
Where are we at with this?  I see that you have copied/"migrated" this to your repository.  Am I holding things up by not turning this .install version into an automatic package?

Some naive questions from a Git noob who doesn't quite "get it" yet...
Do I need to sync your entire repository in order to work on the files locally and submit PRs?
There is no reason to keep the FastCopy.install files in my repository too, right?

**Comment by dtgm** [comment]
>Where are we at with this? 

- [x] Added to my repo.
- [ ] convert fields in nuspec and ps1 files to populate dynamically; easy
- [ ] add xml file detailing where to look and what to retrieve; hard, relatively

The xml is most difficult, but if you want to give it a shot it should be easy since `fastcopy.portable` I already created. Creating these files is a bit of an art because you have to balance how much you assume.

>I see that you have copied/"migrated" this to your repository. Am I holding things up by not turning this .install version into an automatic package?

Not necessarily. I should have already done it :(

>Do I need to sync your entire repository in order to work on the files locally and submit PRs?

Yes. I know git can be confusing at first, so to clarify what is going on:

You can see I referenced this issue in your repo above in the commit message to *my* repo

`https://github.com/dtgm/chocolatey-packages/commit/4d8459d1475e83c0e785cfef15aa9393beb32b30`

If you note the commit message, in full it is

```
Handover fastcopy.install to keep auto-updated with fastcopy and fastcopy.portable

migrated from https://github.com/teknowledgist/Chocolatey-packages/issues/3
```
and shortened on the page as
```
Handover fastcopy.install to keep auto-updated with fastcopy and fastcopy.portable

migrated from teknowledgist/Chocolatey-packages#3
```

>There is no reason to keep the FastCopy.install files in my repository too, right?

Correct. They should be removed. What I should have also done is fork your repo and issue a PR with these files removed referencing their new location in my repo as noted in the [handover](https://github.com/chocolatey/choco/wiki/PackageMantainerHandover) guide.

Then if you wanted to make any changes to the files, you would fork my repo, create a new branch with whatever name makes sense for you. And then create a PR back to my repo's master branch. (Same as you did with Sketchup fix)

And as you probably noticed making additional commits against that branch were auto-added to the PR.

So basically, you want to remove the files and in the commit message write something like:

```
Closes #3 -- fastcopy.install pkg handover

See https://github.com/chocolatey/choco/wiki/PackageMantainerHandover
New location: https://github.com/dtgm/chocolatey-packages
```

`#3` will add that commit as a reference to this issue for easier tracking. Also see https://github.com/dtgm/chocolatey-packages/pull/231#issuecomment-228120076
`Closes` will automatically Close this issue ("Fix" or "Fixes" also closes)
And you want to keep first line to short summary to prevent line continuation.

**Comment by dtgm** [comment]
But to be honest, trying to help with ketarin may be a bit challenging because setup is not completely straightforward which is a kind way of saying it is a clusterfuck of options and config files that need to be coordinated for everything to work together.

This is partly my fault as the ketarin files are not automatically updated in the repo so they could be out of date if I forget to update the repo. I haven't made sure everything is kept updated because I have only received 2 PRs for very minor ketarin fixes in 2+ years.

**Comment by dtgm** [comment]
:\ I looked at https://github.com/dtgm/chocolatey-packages/blob/master/ketarin/fastcopy.portable.xml

It definitely won't be intuitive for you to figure out all what is going on.
